### PR TITLE
SkeletonClient changes, mainly simplification of output formats

### PR DESCRIPTION
**Pull request by kebwi**

### CODE REVIEW

File: caveclient/skeletonservice.py [hunk 1]
```diff
@@ -478,6 +500,18 @@ def get_skeleton(
...
+            if 'vertex_properties' in sk_json.keys():
+                for key in sk_json['vertex_properties'].keys():
+                    # Radius was redundantly store both as a top-level parameter and in vertex_properties.
+                    # We could either check for it (or any such redundancy key) and skip over it, or we could overwrite it.
+                    # Since they were created as duplicates anyway, it doesn't matter which approach is used.
+                    sk_json[key] = sk_json['vertex_properties'][key]
+                del sk_json['vertex_properties']
+            return sk_json
+        if endpoint_format == "flatdict":
+            assert self._server_version >= Version("0.6.0")
```
Comment (fcollman) [inline caveclient/skeletonservice.py:518]: This should raise a more informative exception telling the user to contact their system administrator to ask them to upgrade their skeleton service to at least 0.6.0

Comment (kebwi) [reply caveclient/skeletonservice.py:518]: But this is a true assertion, a verification that the code is performing correctly. The function only receives "dict" and "swc" as inputs (see the function signature). Then on line 475, "dict" is translated to one of two internal formats ("jsoncompressed" or "flatdict") based on the server version. Later on, at the line you indicated in your comment, I am merely confirming (asserting) that the internal format has been assigned correctly based on the server version.

Comment (kebwi) [reply caveclient/skeletonservice.py:518]: @fcollman But this is a true assertion, a verification that the code is performing correctly. The function only receives "dict" and "swc" as inputs (see the function signature). Then on line 475, "dict" is translated to one of two internal formats ("jsoncompressed" or "flatdict") based on the server version. Later on, at the line you indicated in your comment, I am merely confirming (asserting) that the internal format has been assigned correctly based on the server version.

Comment (fcollman) [reply caveclient/skeletonservice.py:518]: got it! make sense didn't read carefully enough

**Pull request merged**